cdc/model/sink.go

```diff
@@ -719,6 +748,115 @@ func (t *SingleTableTxn) GetCommitTs() uint64 {
 	return t.CommitTs
 }
 
+// TrySplitAndSortUpdateEvent split update events if unique key is updated
+func (t *SingleTableTxn) TrySplitAndSortUpdateEvent() error {
+	if len(t.Rows) < 2 {
```

Comment:
is it possible there is only an update event, shall we split it here ?

Reply:
no, it's not needed to split a single update event.

Reply:
What happen if this update event has unique key columns changed ?

Reply:
This method is almost identical to the `convertRowChangedEvents` method. 

After the enable-old-value is removed, this 2 method can be merged into one.

Reply:
this pr focuses on the duplicated key case, the pre-condition is more than two update events emitted.

Reply:
convertRowChangedEvents checks "enable-old-value" and handle key.